packages/maps/src/components/basic/GeoDistanceDropdown.js

```diff
@@ -445,6 +568,7 @@ GeoDistanceDropdown.propTypes = {
 	iconPosition: types.iconPosition,
 	innerClass: types.style,
 	innerRef: types.func,
+	locationField: types.stringOrArray,
```

Comment:
fuzziness and queryFormat props are missing in the prop-types.

(Is the linting check not woking properly?)

Reply:
It doesn't work when we reassign `this.props` to `props` or something. Does it work for you? :thinking:

Reply:
oh yeah, it won't 👍